backend-lib/src/main/rust/lib.rs

```diff
@@ -2759,6 +2759,44 @@ pub extern "C" fn Java_cash_z_ecc_android_sdk_internal_model_TorLwdConn_freeLigh
     }
 }
 
+/// Returns information about this lightwalletd instance and the blockchain.
+#[unsafe(no_mangle)]
+pub extern "C" fn Java_cash_z_ecc_android_sdk_internal_model_TorLwdConn_getServerInfo<'local>(
+    mut env: JNIEnv<'local>,
+    _: JClass<'local>,
+    lwd_conn: jlong,
+) -> jbyteArray {
+    let res = catch_unwind(&mut env, |env| {
+        let lwd_conn = ptr::with_exposed_provenance_mut::<crate::tor::LwdConn>(lwd_conn as usize);
+        let lwd_conn = unsafe { lwd_conn.as_mut() }
+            .ok_or_else(|| anyhow!("A Tor lightwalletd connection is required"))?;
+
+        let info = lwd_conn.get_lightd_info()?;
+
+        Ok(utils::rust_bytes_to_java(env, &info.encode_to_vec())?.into_raw())
```

Comment:
Matching @nuttycom's suggestion on https://github.com/Electric-Coin-Company/zcash-light-client-ffi/pull/202, here (and in all three new methods) I return the protobuf-encoded data. However, in the Android SDK it's rather annoying to do so because I can't _parse_ the data on the Kotlin side because the Rust backend is in `backend-lib` while the Kotlin protobufs are over in `lightwallet-client-lib`. So I think the next necessary step is for someone to write an `sdk-lib` wrapper around `TorLwdConn` that parses the protobufs into the Kotlin types.